### PR TITLE
Updated for 12 week

### PR DESCRIPTION
**Pull request by notadeejay**
var => const

### CODE REVIEW

File: lib/palindromes.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-function isPalindrome(s) {
-  var stringReverse = s.split("").reverse().join("");
+const isPalindrome = function(s) {
+  let stringReverse = s.split("").reverse().join("");
   return s == stringReverse;
 }
```
Comment (kvirani) [inline lib/palindromes.js]: missing semicolon.

**Review by kvirani — CHANGES_REQUESTED** [review]

File: lib/palindromes.js (hunk 1 above)
Comment (kvirani) [inline lib/palindromes.js]:
```suggestion
};
```

**Review by kvirani — CHANGES_REQUESTED** [review]
this time as a suggested edit (new feature in github im not that used to)

File: lib/palindromes.js (hunk 1 above)
Comment (notadeejay) [reply lib/palindromes.js]: ooooh neat

**Pull request merged**